src/main/java/emu/grasscutter/utils/JsonUtils.java

```diff
@@ -44,8 +44,11 @@ public static <T> T decode(JsonElement jsonElement, Class<T> classType) throws J
 
     public static <T> T loadToClass(Reader fileReader, Class<T> classType) throws IOException {
         return gson.fromJson(fileReader, classType);
-    }
 
+
+
+        // return gson.fromJson(fileReader, classType);
+    }
```

Comment:
Remove the whitespace + extra redundant lines.

Reply:
Thank you for your guidance!